.circle/config.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: headspace/hs-node:8.9 # primary container
+        environment:
+          NODE_ENV: test
+          TEST_FAIL_ON_ERROR: true
+          TSLINT_FAIL_ON_ERROR: true
+    working_directory: ~/proxy
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}
+
+      - run:
+          name: Install dependencies
+          command: |
+            echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
+            yarn
+
+      - run:
```

Comment:
do we still need green keeper step? we dont use it anymore

Reply:
chu right. I copy pasta'd